ext/drpttracer/drpttracer.c

```diff
@@ -567,7 +540,7 @@ drpttracer_end_tracing(IN void *drcontext, INOUT void *tracer_handle,
         if (handle->header->data_head > handle->header->data_size) {
             ERRMSG("the buffer is full and the new PT's sideband data is overwritten\n");
             ASSERT(false, "data_head > data_size");
-            return DRPTTRACER_ERROR_FAILED_TO_READ_SIDEBAND_DATA;
+            return DRPTTRACER_ERROR_OVERWRITTEEN_SIDEBAND_DATA;
```

Comment:
s/DRPTTRACER_ERROR_OVERWRITTEEN_SIDEBAND_DATA/DRPTTRACER_ERROR_OVERWRITTEN_SIDEBAND_DATA/

Reply:
And in dox too

Reply:
Done.